server/job-scheduler.js

```diff
@@ -73,7 +75,40 @@ class JobScheduler {
         console.log('------------------------------------------');
 
         const failedJobsList =[];
+        if(dependantJobs.length === 0 && dataflowId){
+          console.log('------------------------------------------');
+          console.log('WORKFLOW EXECUTION COMPLETE, Checking if subscribed for notifications.', )
+          console.log('------------------------------------------');
+          try{
+            const dataflow = await Dataflow.findOne({where : {id : dataflowId}});
+            const cluster = await Cluster.findOne({where : {id : dataflow.clusterId}});
+
+            if(dataflow.dataValues?.metaData?.notification?.notify === 'Always' || dataflow.dataValues?.metaData?.notification?.notify === 'Only on success'){
+              console.log('------------------------------------------');
+              console.log('Prepping to send workflow execution status', )
+              console.log('------------------------------------------');
+              const  hpccURL  = `${cluster.thor_host}:${cluster.thor_port}/#/stub/ECL-DL/Workunits-DL/Workunits`; 
+              const message = `<div>
```

Comment:
Should we move this code to the notification module rather than having the email content embedded with the scheduler?

Reply:
Moved.